[2024-10-17T21:54:49.529+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-17T21:54:49.533+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill_3.taskBranch scheduled__2024-10-12T00:00:00+00:00 [queued]>
[2024-10-17T21:54:49.535+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill_3.taskBranch scheduled__2024-10-12T00:00:00+00:00 [queued]>
[2024-10-17T21:54:49.535+0100] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-17T21:54:49.538+0100] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): taskBranch> on 2024-10-12 00:00:00+00:00
[2024-10-17T21:54:49.541+0100] {standard_task_runner.py:72} INFO - Started process 6258 to run task
[2024-10-17T21:54:49.543+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill_3', 'taskBranch', 'scheduled__2024-10-12T00:00:00+00:00', '--job-id', '313', '--raw', '--subdir', 'DAGS_FOLDER/cron_catchup_backfill_3.py', '--cfg-path', '/var/folders/nl/1x330z2x7yzb268yb0bbvjcr0000gn/T/tmpq8u09tk1']
[2024-10-17T21:54:49.545+0100] {standard_task_runner.py:105} INFO - Job 313: Subtask taskBranch
[2024-10-17T21:54:49.557+0100] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill_3.taskBranch scheduled__2024-10-12T00:00:00+00:00 [running]> on host mujtabas-macbook-pro.local
[2024-10-17T21:54:49.576+0100] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ali' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill_3' AIRFLOW_CTX_TASK_ID='taskBranch' AIRFLOW_CTX_EXECUTION_DATE='2024-10-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-12T00:00:00+00:00'
[2024-10-17T21:54:49.577+0100] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-17T21:54:49.581+0100] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-17T21:54:49.583+0100] {python.py:240} INFO - Done. Returned value was: taskC
[2024-10-17T21:54:49.583+0100] {branch.py:38} INFO - Branch into taskC
[2024-10-17T21:54:49.584+0100] {skipmixin.py:230} INFO - Following branch {'taskC'}
[2024-10-17T21:54:49.586+0100] {skipmixin.py:278} INFO - Skipping tasks [('taskE', -1)]
[2024-10-17T21:54:49.591+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-17T21:54:49.592+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill_3, task_id=taskBranch, run_id=scheduled__2024-10-12T00:00:00+00:00, execution_date=20241012T000000, start_date=20241017T205449, end_date=20241017T205449
[2024-10-17T21:54:49.605+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-17T21:54:49.614+0100] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-17T21:54:49.614+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
